test/language/module-code/dynamic-import/usage/nested-if-braceless-eval-gtbndng-indirect-update-dflt.js

```diff
@@ -0,0 +1,37 @@
+// This file was procedurally generated from the following sources:
+// - src/dynamic-import/eval-gtbndng-indirect-update-dflt.case
+// - src/dynamic-import/default/nested-if-braceless.template
+/*---
+description: Modifications to default binding that occur after dependency has been evaluated are reflected in local binding (nested if syntax)
+esid: sec-import-call-runtime-semantics-evaluation
+features: [dynamic-import]
+flags: [generated, async]
+info: |
+    ImportCall :
+        import( AssignmentExpression )
+
+    1. Let referencingScriptOrModule be ! GetActiveScriptOrModule().
+    2. Assert: referencingScriptOrModule is a Script Record or Module Record (i.e. is not null).
+    3. Let argRef be the result of evaluating AssignmentExpression.
+    4. Let specifier be ? GetValue(argRef).
+    5. Let promiseCapability be ! NewPromiseCapability(%Promise%).
+    6. Let specifierString be ToString(specifier).
+    7. IfAbruptRejectPromise(specifierString, promiseCapability).
+    8. Perform ! HostImportModuleDynamically(referencingScriptOrModule, specifierString, promiseCapability).
+    9. Return promiseCapability.[[Promise]].
+
+
+    GetBindingValue (N, S)
+
+    [...]
+    3. If the binding for N is an indirect binding, then
+       a. Let M and N2 be the indirection values provided when this binding for
+          N was created.
+       b. Let targetEnv be M.[[Environment]].
+       c. If targetEnv is undefined, throw a ReferenceError exception.
+       d. Let targetER be targetEnv's EnvironmentRecord.
+       e. Return ? targetER.GetBindingValue(N2, S).
+
+---*/
+
+if (true) import('./eval-gtbndng-indirect-update-dflt_FIXTURE.js');
```

Comment:
☝️ Based on [INTERPRETING](https://github.com/tc39/test262/blob/377f07f502890284d1e89ecb6b66343464fe14c0/INTERPRETING.md) async flagged tests should print `'Test262:AsyncTestComplete'` or `'Test262:AsyncTestFailure: ' + reason` by way of [`doneprintHandle`](https://github.com/tc39/test262/blob/master/harness/doneprintHandle.js). Without print a complete or fail the interpretation can hit a test timeout and interpret it as a failure.

Reply:
Nice catch! I think these are all from my original batch of generated tests, which definitely need work. Please keep the feedback and review coming <3